package.json

```diff
@@ -55,7 +54,8 @@
     "tsconfig-paths-jest": "^0.0.1",
     "tsx": "^4.19.2",
     "typescript": "^5.7.2",
-    "typescript-eslint": "^8.16.0"
+    "typescript-eslint": "^8.16.0",
+    "uuid": "^11.0.3"
```

Comment:
We do actually need `uuid`, because of `.yalc/shared/graphql/clientIdHeader.ts`. One of those `yalc` quirks.